csob/api.py

```diff
@@ -272,7 +300,13 @@ def echo(self, method: HTTPMethod = HTTPMethod.GET) -> APIResponse:
         Returns:
             APIResponse
         """
-        raise NotImplementedError()
+        resource = EchoResource(**self.get_resource_kwargs())
```

Comment:
proč je echo na post i get?

Reply:
Aby se otestoval jak způsob předavání GET parametru (`?signature=sdafds...`) tak i POST (JSON)

Reply:
To pak alenení echo. Echo je jen obecný test, že je cíl na příjmu. Takto to mají definované oni?

Reply:
jo to tam mají nadefinované oni